### PR TITLE
Ignore pipes on unix

### PR DESCRIPTION
**Pull request by Jake-Shadle**
Resolves: #11

### CODE REVIEW

File: src/licenses/mod.rs [hunk 1]
```diff
@@ -216,6 +216,17 @@ impl Gatherer {
     }
 }
 
+#[cfg(unix)]
+fn is_fifo(ft: std::fs::FileType) -> bool {
+    use std::os::unix::fs::FileTypeExt;
+    ft.is_fifo()
+}
+
+#[cfg(not(unix))]
```
Comment (MaikKlein) [inline src/licenses/mod.rs]: Why does this exist? `is_fifo` only seems to be used in an a `cfg(unix)` block.

Comment (MaikKlein) [reply src/licenses/mod.rs]: We might also just want to remove the `cfg(unix)` block below.

Comment (Jake-Shadle) [reply src/licenses/mod.rs]: Because is_fifo is only available on unix, not windows.

Comment (MaikKlein) [reply src/licenses/mod.rs]: Oh I see you actually made an if block below and not cfg'ed out the whole block.

Comment (Jake-Shadle) [reply src/licenses/mod.rs]: Actually that would be better

**Pull request merged**